var/spack/repos/builtin/packages/linux-pam/package.py

```diff
@@ -27,6 +27,11 @@ class LinuxPam(AutotoolsPackage):
     depends_on("automake", type="build")
     depends_on("libtool", type="build")
 
+    def flag_handler(self, name, flags):
+        if name == "ldflags":
+            flags += ["-lintl"]  # Addresses https://github.com/spack/spack/issues/44637
+        return super().flag_handler(name, flags)
```

Comment:
```suggestion
        return (flags, None, None)
```

`AutoToolsPackage` doesn't set any special `flag_handler` so I'd suggest instead returning the standard output described the docs. https://spack.readthedocs.io/en/latest/packaging_guide.html#compiler-flags